gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -231,54 +233,71 @@ def eink_st_sonderausgaben_bis_2011(
     return out
 
 
-def eink_st_sonderausgaben_ab_2012(
+@add_rounding_spec(params_key="eink_st_abzuege")
+def sonderausgaben_betreuung(
+    eink_st_abzuege_params: dict,
+    anz_kinder_bis_13_hh: int,
     betreuungskost_m: float,
-    kind: bool,
-    anz_kinder_tu: int,
     anz_erwachsene_tu: int,
-    eink_st_abzuege_params: dict,
-    gemeinsam_veranlagt_tu: bool,
 ) -> float:
-    """Calculate sonderausgaben for childcare since 2012.
+    """Calculate sonderausgaben for childcare for childen under 14.
 
     We follow 10 Abs.1 Nr. 5 EStG. You can
     details here https://www.buzer.de/s1.htm?a=10&g=estg.
     Parameters
     ----------
     betreuungskost_m
         See basic input variable :ref:`betreuungskost_m <betreuungskost_m>`.
-    kind
-        See basic input variable :ref:`kind <kind>`.
-    anz_kinder_tu
-        See :func:`anz_kinder_tu`.
+    anz_kinder_bis_13_hh
```

Comment:
This should be based on the tax unit, i.e. `anz_kinder_bis_13_tu`

Reply:
You're right! I changed the aggregation in 'aggregation_demographic_vars'.